bootstrap/sql/migrations/native/1.2.0/mysql/schemaChanges.sql

```diff
@@ -186,4 +186,36 @@ CREATE TABLE IF NOT EXISTS doc_store (
     INDEX doc_store_name_index(name)
 );
 
+CREATE TABLE IF NOT EXISTS installed_application (
```

Comment:
lets use "apps" consistently and we can call it as installed_apps

Reply:
updated